src/imitation/algorithms/bc.py

```diff
@@ -30,17 +29,16 @@ class BCTrainer:
   def __init__(self,
                env,
                *,
-               expert_rollouts: Tuple[np.ndarray, ...],
+               expert_demos: rollout.Transitions,
                policy_class: Type[ActorCriticPolicy] = FeedForward32Policy,
                batch_size: int = 32):
     self.env = env
     self.policy_class = policy_class
     self.batch_size = batch_size
-    expert_obs, expert_acts, _, _ = expert_rollouts
     self.expert_dataset = Dataset(
         {
-            "obs": expert_obs,
-            "act": expert_acts,
+            "obs": expert_demos.old_obs,
```

Comment:
This is confusing naming. If `old_obs`, why not `old_act`? Going to add comment on the `Transitions` class below.